Libraries/vendor/core/_wrapObjectFreezeAndFriends.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @author Ben Newman (@benjamn) <ben@benjamn.com>
```

Comment:
Can you enable Prettier for this file with `@format`?

Reply:
Sure.